lightning-block-sync/src/http.rs

```diff
@@ -129,12 +122,11 @@ impl HttpClient {
 	/// Returns the response body in `F` format.
 	#[allow(dead_code)]
 	pub async fn get<F>(&mut self, uri: &str, host: &str) -> std::io::Result<F>
-	where F: TryFrom<Vec<u8>, Error = std::io::Error> {
-		let request = format!(
-			"GET {} HTTP/1.1\r\n\
-			 Host: {}\r\n\
-			 Connection: keep-alive\r\n\
-			 \r\n", uri, host);
+	where
+		F: TryFrom<Vec<u8>, Error = std::io::Error>,
+	{
+		let request =
+			format!("GET {} HTTP/1.1\r\nHost: {}\r\nConnection: keep-alive\r\n\r\n", uri, host);
```

Comment:
Yeah, I don't mind the automatic wrapping. But it's weird that it unwraps something because it can fit on a line. Maybe we should just use `#[rustfmt::skip]` in those cases?